controllers/trainingjob/trainingjob_controller.go

```diff
@@ -99,6 +99,12 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 
 	switch ctx.TrainingJob.Status.TrainingJobStatus {
 	case string(sagemaker.TrainingJobStatusCompleted):
+		// If there are any debug rule job still in progress
```

Comment:
nit: Can we flesh out this comment? `If there are any debug rule jobs still in progress, keep requeueing.`